.travis.yml

```diff
@@ -1,3 +1,8 @@
+sudo: required
```

Comment:
I like some of CircleCI's features, but I'm also fine with using Travis. If you're happy then let's go ahead and merge. :)